blog/_posts/2023-09-28-t31-set-discussion.md

```diff
@@ -0,0 +1,76 @@
+last_update: 28/09/2023
+excerpt: What does the new tier set do and how does it work?
+author: Elivrio
+toc: true
+---
+
+## T31 effects
+> (2) Set Bonus: {{ site.data.talent.primordial_wave }} also casts an {{ site.data.talent.eb }} at the target at 100% effectiveness, and grants you all three {{ site.data.talent.eb }} bonus effects for 15.0 seconds.
+
+> (4) Set Bonus: {{ site.data.spell.lvb }} burns your target for 15% additional damage over 4 seconds, and after consuming {{ site.data.talent.primordial_wave }}, your next 3 casts of {{ site.data.spell.lvb }} also hit up to 3 additional targets affected by your {{ site.data.spell.fs }} for 100% of normal damage.
+
+First things first. Without being talented into {{ site.data.talent.primordial_wave }} these bonuses do nothing.
+Which means, we're forced into taking this talent for the next tier.
+
+The reworked 4p also puts all of its emphasis on {{ site.data.spell.lvb }}, killing the "weaving" we expected with the first iterations.
+
+<hr>
+
+## 2p
+The additional damage from {{ site.data.talent.eb }} on {{ site.data.talent.primordial_wave }} is a great way to improve the immediate value and feel of the talent.
+While applying {{ site.data.spell.fs }} is a good feature to have, adding the damage makes casting it so much more rewarding.
+
+Furthermore this effect grants all three {{ site.data.talent.eb }} buffs. These present a sizeable chunk of stats.
+
+The additional {{ site.data.talent.eb }} is cast for free. And there exist multiple talents triggered by casting {{ site.data.talent.eb }}.
+Here is a table of them and whether they are triggered by the 2p effect.
+
+Talent | 2p can trigger it
+--- | ---
+{{ site.data.talent.sop }} | ❌ no
+{{ site.data.talent.lr }} | ✅ yes
+{{ site.data.talent.mwf }} | ✅ yes
+{{ site.data.talent.fb }} | ✅ yes
+{{ site.data.talent.wlr }} | ✅ yes
+
+<hr>
+
+## 4p
+
+Any kind of weaving is now gone and you just want to press {{ site.data.spell.lvb }} all day everyday.
+
+Concerning the burn added to {{ site.data.spell.lvb }}, Molten Slag, it functions similarly to Mage's [ignite](https://www.wowhead.com/spell=12654/ignite). It stacks and extends itself if another {{ site.data.spell.lvb }} is used while the target is burning. That's pretty neat and you don't have to play around the debuff duration.
+
+The recent changes to the 4p are good enough to use {{ site.data.spell.lvb }} even when more than 4 targets are present,
+
+<hr>
+
+## Issues
+
+#### 2p Bugs
+1. The 15s {{ site.data.talent.eb }} buff you get from 2p gets overrides by any new {{ site.data.talent.eb }} cast.
+2. The free {{ site.data.talent.eb }} from {{ site.data.talent.pw }} does not trigger {{ site.data.talent.sop }}.
+
+<br>
+#### 4p Bugs
+1. The first 3 {{ site.data.spell.lvb }} you cast after {{ site.data.talent.pw }} will also hit up to 3 additional targets. Currently, this works in ST and you get 1 additional {{ site.data.spell.lvb }} that can overload for each of those 3 casts.
```

Comment:
I find this phrasing confusing. Here is my take on it:

1. Your main target is currently considered an "additional target" for the sake of the 4-piece bonus. This means that in Single Target the first three {{ site.data.spell.lvb }} after each {{ site.data.talent.pw }} will be duplicated, with each being able to overload.